src/mammos_dft/db.py

```diff
@@ -270,23 +274,42 @@ def find_unique_material(print_info=True, **kwargs):
 
 
 def get_cif(
-    short_label=None,
-    chemical_formula=None,
-    space_group_name=None,
-    space_group_number=None,
-    cell_length_a=None,
-    cell_length_b=None,
-    cell_length_c=None,
-    cell_angle_alpha=None,
-    cell_angle_beta=None,
-    cell_angle_gamma=None,
-    cell_volume=None,
-    ICSD_label=None,
-    OQMD_label=None,
-    print_info=True,
-    outdir="out",
-):
-    """Load cif and move it to the directory `outdir`."""
+    short_label: str | None = None,
+    chemical_formula: str | None = None,
+    space_group_name: str | None = None,
+    space_group_number: int | None = None,
+    cell_length_a: float | None = None,
+    cell_length_b: float | None = None,
+    cell_length_c: float | None = None,
+    cell_angle_alpha: float | None = None,
+    cell_angle_beta: float | None = None,
+    cell_angle_gamma: float | None = None,
+    cell_volume: float | None = None,
+    ICSD_label: str | None = None,
+    OQMD_label: str | None = None,
+    print_info: bool = True,
+    outdir: str | pathlib.Path = "out",
+) -> None:
+    """Load cif and move it to the directory `outdir`.
+
+    Args:
+        short_label:  (Default value = None)
+        chemical_formula:  (Default value = None)
+        space_group_name:  (Default value = None)
+        space_group_number:  (Default value = None)
+        cell_length_a:  (Default value = None)
+        cell_length_b:  (Default value = None)
+        cell_length_c:  (Default value = None)
+        cell_angle_alpha:  (Default value = None)
+        cell_angle_beta:  (Default value = None)
+        cell_angle_gamma:  (Default value = None)
+        cell_volume:  (Default value = None)
+        ICSD_label:  (Default value = None)
+        OQMD_label:  (Default value = None)
+        print_info:  (Default value = True)
+        outdir:  (Default value = "out")
```

Comment:
@andrea-petrocchi This is autogenerated (and useless); can I copy the docstring from one of the other functions?